activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -801,8 +809,8 @@ def mismatched_foreign_key(message, sql:, binds:)
           MismatchedForeignKey.new(options)
         end
 
-        def version_string
-          full_version.match(/^(?:5\.5\.5-)?(\d+\.\d+\.\d+)/)[1]
+        def version_string(full_version_string) # :nodoc:
```

Comment:
Private method is not to be exposed in the doc, so the `# :nodoc:` is redundant.